src/main/java/com/example/spring_bank_project/account/domain/model/StandardAccount.java

```diff
@@ -0,0 +1,14 @@
+package com.example.spring_bank_project.account.domain.model;
+
+import com.example.spring_bank_project.account.domain.valueObject.AccountId;
+import com.example.spring_bank_project.account.domain.valueObject.AccountType;
+import lombok.Value;
+
+import java.time.Instant;
+
+@Value
+public class StandardAccount {
+    AccountId accountId;
+    AccountType type;
```

Comment:
skoro jest juz typem okreslone jako StandardAccount to po co jeszcze robic pole AccountType? powinno ono tez chyba odrazu przyjmowac type = Standard aby nie dalo sie zrobic np StandardAccount z polem typu AccountType = NON_STANDARD (zakladajac ze w przyszlosci bylby taki dodany) (zakladajac ze taka byla z tym rozkmina)

Reply:
yup masz rację